snapcraft/plugins/autotools.py

```diff
@@ -98,6 +98,8 @@ def build(self):
                 path = os.path.join(self.builddir, script)
                 if not os.path.exists(path):
                     continue
+                if os.path.isdir(path):
```

Comment:
I would just put this in the `if` above but not going to be dramatic about it ;-)